frontend/src/lib/components/Admin/EditMilestoneModal.svelte

```diff
@@ -75,6 +76,10 @@ export async function saveChanges() {
 				{/each}
 			</div>
 		{/each}
+		<div class="mb-5">
+		<Label>{`${$_("admin.expected-age")}: ${milestone.expected_age_months}m`}</Label>
+		<Range id="expected-age-months" min="1" max="72" bind:value={milestone.expected_age_months} />
```

Comment:
when I add a milestone and then try to edit it, this doesn´t show up because the `milestone` variable is still `null`. Didn´t track down what goes wrong there in detail though...